app/views/messages/_message.html.erb

```diff
@@ -1,6 +1,7 @@
  <div class="message-container" id="message-<%= message.id %>">
       <i class="author">
-        <span><%= message.user.email %></span>
+        <span><%= message.user.username.capitalize %></span></br>
```

Comment:
usernames are usually all lower case. So I would delete the capitalize

Reply:
I can do ... but here it wasnt capitalized ...